src/main/webapp/view/discussion/discussion-area.js

```diff
@@ -40,32 +39,60 @@ export default class DiscussionArea {
     this.#lecture = lecture;
     this.#manager = new DiscussionManager(this.#lecture);
     this.#currentTimeMs = 0;
-    this.#currentRootCommentElements = [];
     this.#nearestComments = [];
   }
 
   /**
    * Initialize the discussion area by loading the current comments.
    */
   async initialize() {
-    await this.loadDiscussion();
+    await this.updateDiscussion();
   }
 
   /**
-   * Fetches and displays the current comments.
+   * Fetches and updates the currently displayed comments.
    */
-  async loadDiscussion() {
-    // Clear any existing comments before loading.
-    ELEMENT_DISCUSSION.textContent = '';
-    this.#currentRootCommentElements = [];
-
-    const rootComments = await this.#manager.fetchRootComments();
-    for (const rootComment of rootComments) {
-      const rootCommentElement = new DiscussionComment(this);
-      rootCommentElement.setComment(rootComment);
-      this.#currentRootCommentElements.push(rootCommentElement);
-      ELEMENT_DISCUSSION.appendChild(rootCommentElement);
+  async updateDiscussion() {
+    const newComments = await this.#manager.fetchNewComments();
+    if (newComments.length == 0) {
+      return;
     }
+
+    for (const comment of newComments) {
```

Comment:
Can these two for loops be merged into one?

Reply:
Unfortunately not.  I need to assign `.element` to all comments first, because the second loop references the parent's element, and that needs to be defined.